sim_pipeline/image_simulation.py

```diff
@@ -71,17 +71,17 @@ def gsobj(image, pix_scale, flux):
     :param pix_scale: pixel scale to be asigned to the interpolated image
     :param flux: flux value to be asigned to the interpolated image
     :returns : interpolated image with specified pixel scale and flux"""
-    gso = galsim.InterpolatedImage(galsim.Image(image), scale = pix_scale, flux = flux)
```

Comment:
I would not change these parts of the code since @nkhadka21 is working on and with them

Reply:
Got it! Will find a work around to the galsim installation issues.